person/views.py

```diff
@@ -132,6 +133,8 @@ def get_context_data(self, **kwargs):
                     commander_end = Q(object_ref__membershippersonlastciteddate__value__gte=mem_start)
                     end_is_null = Q(object_ref__membershippersonlastciteddate__value__isnull=True)
 
+                    last_cited_attr = '-object_ref__membershippersonlastciteddate'
```

Comment:
So does the `order_by` work in this case? Or should we also change the following ORM query to raw SQL?

Reply:
To be clearer: by "in this case" I mean in the [big query block that follows](https://github.com/security-force-monitor/sfm-cms/blob/fa91ffe14d438c8696000cce06418d3d38d24c95/person/views.py#L126-L143).

Reply:
Ah, yes. We likely need the raw SQL to generate `commanders`.